src/main/java/com/newrelic/experts/jenkins/EventRecorderImpl.java

```diff
@@ -292,4 +291,31 @@ public void setCustomAttributes(
       }
     }
   }
+
+  @Override
+  public void recordJenkinsMasterEvent(JenkinsMasterEvent jenkinsMasterEvent) {
```

Comment:
@dalvizu per above can we either stick with just passing in the two parameters here (`isQuietDownMode` and `agentConnectedCount`)?